lib/os.js

```diff
@@ -54,3 +54,6 @@ if (binding.isBigEndian)
   exports.endianness = function() { return 'BE'; };
 else
   exports.endianness = function() { return 'LE'; };
+
+exports.sizeof = process.binding('sizeof');
+exports.alignof = process.binding('alignof');
```

Comment:
`lib/os.js` also is only executed once (and the exports are cached), so yes, the values should basically be cached indefinitely.